packages/astro/components/Code.astro

```diff
@@ -23,6 +23,12 @@ interface Props extends Omit<HTMLAttributes<'pre'>, 'lang'> {
 	 * @default "plaintext"
 	 */
 	lang?: BuiltinLanguage | SpecialLanguage | LanguageRegistration;
+	/**
+	 * A metastring to pass to the highlighter.
+	 * 
+	 * @default undefined
+	 */
+	meta?: string;
```

Comment:
Can we also add a link here to https://shiki.style/guide/transformers#meta? That way others know what this meta is about.